.github/workflows/update-flake.yml

```diff
@@ -41,22 +41,28 @@ jobs:
             email "$id+$name@users.noreply.github.com" \
             >>"$GITHUB_OUTPUT"
 
-      - uses: DeterminateSystems/update-flake-lock@v26
+      - name: setup git
+        env:
+          name: ${{ steps.user-info.outputs.name }}
+          email: ${{ steps.user-info.outputs.email }}
+        run: |
+          git config --global user.name $name
+          git config --global user.email $email
+
+      - name: update lock files
+        run: |
+          nix flake update \
+            --commit-lock-file \
+            --option commit-lock-file-summary "flake: update root inputs"
+
+          nix flake update \
+            --commit-lock-file \
+            --flake ./flake/dev \
+            --option commit-lock-file-summary "flake: update dev inputs"
+
+      - uses: peter-evans/create-pull-request@v7
```

Comment:
It'd be better to avoid depending on an external action; this is trivial with `gh pr create`